link-user-to-episys-name-record/REPWRITERSPECS/BANNO.DATABASE.CHECK.V1.POW

```diff
@@ -480,6 +642,7 @@ SUCCESS
            IF MBRLINKFOUND=TRUE OR
```

Comment:
Overall the improvements to the code are very helpful to have Episys Notes and Debug Code to see in the Logs! Thank you for all the work Tom. 

I think I found a minor issue in the logic of part of Scenario 3.
I think you mean "IF MBRLINKFOUND=TRUE **AND** ADDRLINKFOUND=TRUE THEN"  Instead of OR (meaning one name record has match one and another name record has match for the other) on line 642 for LINKTRACKING Logic Scenario #3.c.2, as The conditions statements above where it "says 3.c.1. If both numbers are found in a single name record, return success." 

Otherwise I think none of the code below #3.c.2 would happen, which is designed to handle the situation of having found only as single name that either contains the matching MBRLINK or the matching ADDRLINK.

I also, suggest promoting scenario 3 to the top of LINKSTATE, as 3.c.1 not needing to update tracking 8, would eventually be the most common scenario, unless the more common scenario is Members with both a Prime and Joint Name, making the error for multiple names .

Please take look at the other comments/suggestions I left that are not part of the code review.
Let me know if I am mistaken somehow about the rest of the Scenarios in 3 after #3.c.2 not being able to happen.

Thank you,

Reply:
Jordan.... I'm not sure about your comment on 3.c.1 & 3.c.2.

3.c.1 is looking for a single name record where both the member link and the address link match. IF there is no name record where both the name and address links match then the next check is 3.c.2:  the program then looks to see if there is at least one record where either the address link OR the name link matches.  Call me if you'd like to discuss.